integration-libs/s4-service/order/components/s4-service-order-detail-actions/s4-service-order-detail-actions.component.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { OrderDetailActionsComponent } from '@spartacus/order/components';
+import { Order } from '@spartacus/order/root';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-s4-service-order-detail-actions',
+  templateUrl: './s4-service-order-detail-actions.component.html',
+})
```

Comment:
```suggestion
@Component({
  selector: 'cx-s4-service-order-detail-actions',
  templateUrl: './s4-service-order-detail-actions.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush,
})
```
major: let's keep the convention and add change detection strategy OnPush to prevent possible redundant re-rendering of this component and its child.